Patches.py

```diff
@@ -42,6 +42,31 @@
 OverrideEntry: TypeAlias = "tuple[int, int, int, int, int, int]"
 
 
+def patch_max_upgrade_flags(spoiler: Spoiler, world: World, rom: Rom):
+    # This list must have the same elements and order as the ProgressiveItemIdentifier enum
+    max_upgrade_items = [
+        "Hookshot",
+        "Strength",
+        "Bomb Bag",
+        "Bow",
+        "Slingshot",
+        "Wallet",
+        "Scale",
+        "Nut Capacity",
+        "Stick Capacity",
+        "Bombchus",
+        "Magic Meter",
+        "Ocarina",
+    ]
```

Comment:
Then is there a reason this isn't also an enum?

Reply:
Not really, but due to changes requested on discord the stuff on Patches.py is no longer required